cypress/support/commands.js

```diff
@@ -0,0 +1,11 @@
+const compareSnapshotCommand = require('cypress-visual-regression/dist/command');
+const beforeCompareSnapshotCommand = require('./commands/beforeCompareSnapshotCommand');
+
+// noinspection JSCheckFunctionSignatures
+compareSnapshotCommand({
+    "errorThreshold": 0.01,
```

Comment:
This threshold is necessary, because of color profile difference between my personal linux environment and CI instance on Github actions. Maybe you have an idea, howto to avoid these threshold.